package.json

```diff
@@ -22,6 +22,9 @@
     "start": "spandx",
     "lerna": "lerna",
     "test": "node scripts/test.js",
+    "test:watch": "npm config set group=default && web-test-runner --node-resolve --playwright --watch --group=$npm_config_group",
```

Comment:
Using `$npm_config_group` let's us pass commands via arguments like "npm run test:watch --group with-vue".

In this case `--group` maps to `$npm_config_group` which means `$npm_config_group == with-vue`

We use `npm config set group=default` to set a default that we can override so `$npm_config_group == default` if no arguments are passed in.

More here: https://docs.npmjs.com/cli/v7/using-npm/config#environment-variables

Reply:
Love this approach; stealing this to use everywhere :P